.github/workflows/ci.yaml

```diff
@@ -0,0 +1,51 @@
+name: Dart CI
+
+on:
+  push:
+    branches:
+      - 'master'
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup working Directory
+        run: |
+          sudo mkdir -p /usr/src/extension
+          sudo cp -R ./* /usr/src/extension
+          cd /usr/src/extension
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '17'
+
+      - name: Install NVM
+        run: |
+          curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | bash
+          . ~/.nvm/nvm.sh
+          nvm install 17
+
+      - name: Install Dependencies
+        run: |
+          sudo apt install -y gzip tar
+          sudo apt update -y
+          sudo apt upgrade -y
```

Comment:
Since we're in GHA now and the runners are kept up to date, I think we can remove the nvm install and dep updates
```suggestion
```